.travis.yml

```diff
@@ -11,3 +13,10 @@ install:
 
 script:
   - make flake
+  - make test
+
+before_script:
+  # Disable services enabled by default
+  # http://docs.travis-ci.com/user/database-setup/#MySQL
+  - sudo /etc/init.d/postgresql stop
```

Comment:
Why bother?

Reply:
Port conflict, tests expose PG port to localhost

Reply:
😞 That's one of the real problems with current testing practices.